documentation/src/main/asciidoc/topics/proc_enabling_jmx_port.adoc

```diff
@@ -11,23 +11,43 @@ If you want to remotely access {brandname} Server via JMX you must enable a remo
 ====
 endif::remote_caches[]
 
-.Procedure
+Enabling remote JMX involves exposing sensitive information about your system.
+You can enable remote JMX connections using one of the following approaches:
+
+* Enable remote JMX connections with authentication.
```

Comment:
I think it should be "remote JMX ports" not connections. You can enable JMX through the single port `11222` if you want to. Or, if you want a dedicated port for JMX that is more like what you get with WildFly, then you can expose JMX on `9999`.